logging.rst

```diff
@@ -308,6 +308,15 @@ option of your handler to ``rotating_file``:
             ),
         ));
 
+Injecting a Logger into a Service
+----------------------------------
+
+To use a logger in your own service, inject it using Dependency Injection
+with ``@logger``. The tag ``monologlog.logger`` tag must be used. If you want to
```

Comment:
I find this phrase confusing: `The tag monologlog.logger tag must be used.` It looks like you must add that tag to the service where you use `@logger`.